utils/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { IGameWeek, IUser, IWeeklyPicks } from '@/api/IapiFunctions';
+import { getAllWeeklyPicks, getCurrentGame } from '@/api/apiFunctions';
+import { Models } from 'appwrite/types/models';
+import { clsx, type ClassValue } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs));
+}
+
+export const getGameData = async (
+  userId: IUser['id'],
```

Comment:
nit: Make these into an object instead. Easier to deconstruct objects than ensuring the order of props are correct.

Reply:
✅ Done.